src/DOM.ts

```diff
@@ -298,4 +298,33 @@ export class DOM {
     }
     return set;
   }
+
+
+  /**
+   * Heartland.DOM.addCertAlert
+   *
+   * Adds an alert letting the developer know they're in sandbox mode
+   *
+   * @param {Heartland.HPS} hps
+   * @param {string} elementid
+   */
+  public static addCertAlert() {
+
+    const el = document.createElement('div');
+    const text = document.createTextNode("You're in test mode.");
+    
+    el.appendChild(text);   
+
+    el.setAttribute("style", "display:block;width:100%;margin-bottom:5px;color:#fff;background-color: #770000;padding: 8px 5px;font-family: Verdana;font-weight: 100;font-size: 12px;text-align: center;box-sizing: border-box;");
```

Comment:
It might be better to leverage the JavaScript style API to set these values instead of one big giant string. Here's an example:

```typescript
el.style.display = "block";
el.style.width = "100%";
el.style.marginBottom = "5px";
// ...
```